debian/copyright

```diff
@@ -1,19 +1,21 @@
 Format: http://dep.debian.net/deps/dep5
 Upstream-Name: nest-simulator
-Source: https://github.com/nest/nest-simulator/archive/master.zip
+Source: https://github.com/nest/nest-simulator/
 
-Files: debian/*
-Copyright: 2019 The NEST Initiative <users@nest-simulator.org>
-License: GPL-2
+Files: *
+Copyright: 2004 The NEST Initiative
+License: GPL-2+
+
+License: GPL-2+
```

Comment:
This line looks redundant to me and I don't understand why the block below is indented. But that might all be just because I don't know anything about the syntax of this file.

Reply:
It is a syntax thing.

Reply:
Alright, then forget about my comment.